Dockerfile

```diff
@@ -0,0 +1,52 @@
+# hadolint global ignore=DL3008
+FROM debian:buster
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+      python \
+      python-pip \
+      python-dev \
+      python-setuptools \
+      ca-certificates \
+      gcc \
+      libpq-dev \
+      curl \
+      gunicorn \
+    && rm -rf /var/lib/apt/lists/*
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# kelseyhightower confd arm64 builds are broken: https://github.com/kelseyhightower/confd/issues/850
+ARG CONFD_VERSION=0.19.1
+RUN CONFD_ARCH=$(arch | sed s/aarch64/arm64/ | sed s/x86_64/amd64/) \
+    && curl -SL "https://github.com/abtreece/confd/releases/download/v${CONFD_VERSION}/confd-v${CONFD_VERSION}-linux-${CONFD_ARCH}.tar.gz" | tar -xz -C /usr/local/bin/ \
+    && /usr/local/bin/confd -version
+
+RUN groupadd --system --gid 510 osqa && useradd --system --gid osqa --uid 510 osqa
+
+WORKDIR /usr/src/app
+RUN chown osqa:osqa /usr/src/app
+
+COPY requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY --chown=osqa . .
```

Comment:
Why are we copying the current directory to itself? Is this just a long winded way of writing `chown` or something? How did any files get into this directory anyway?

Reply:
This is to `COPY` the files from current working directory (outside the container, `.` context is the Dockerfile path) into the container's `WORKDIR`.

The `--chown` sets the owner within the container to allow python to create `.pyc` cache files.